schema.sql

```diff
@@ -0,0 +1,16 @@
+CREATE DATABASE vet_clinic
+    WITH 
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    CONNECTION LIMIT = -1;
+
+CREATE TABLE IF NOT EXISTS public.animals(
+	Id SERIAL PRIMARY KEY,
+	name VARCHAR,
+	date_of_birth DATE,
+	escape_attempts INT DEFAULT 0,
+	neutered BOOLEAN,
+    weight_kg DECIMAL
+);
+
+SELECT * FROM animals;
```

Comment:
- [ ] Let's move this query to the queries.sql file because everything that is a query that helps to analyze data should be saved in the queries.sql file to make our work look professional and more organised.